src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCacheOptions.cs

```diff
@@ -53,6 +53,12 @@ public double CompactionPercentage
             }
         }
 
+        /// <summary>
+        /// Gets or sets the flat that controls linked entries tracking. Disabled by default.
+        /// </summary>
+        /// <remarks>Prior to .NET 7 this feature was always enabled.</remarks>
```

Comment:
Does that mean this is a breaking change?  Above it says it's disabled by default but here it says it was previously always enabled.

Reply:
> Does that mean this is a breaking change? Above it says it's disabled by default but here it says it was previously always enabled.

Yes, exactly. I've added `needs-breaking-change-doc-created` to ensure we document that.

Reply:
This has been discussed and approved at the API review: https://github.com/dotnet/runtime/issues/45592#issuecomment-896214586